content/2.2database.md

```diff
@@ -221,4 +221,4 @@ the language/library takes care to identify the exception type for us and return
 #### Links
 
 - Previous section: [Basic Functionality](content/2.1functionality.md)
-- Next section: 
\ No newline at end of file
+- Next section: 
```

Comment:
No worries :smile: I wrote this text on Jan 1st and I despite being a Grammar Nazi, I couldn't just review the stuff :stuck_out_tongue_winking_eye: